src/test/java/io/appium/java_client/ios/XCUIAutomationTest.java

```diff
@@ -51,6 +52,18 @@
         }
     }
 
+    @Test public void testPutIntoBackgroundAndRestore() {
+        final long msStarted = System.currentTimeMillis();
+        driver.runAppInBackground(4);
+        assertThat(System.currentTimeMillis() - msStarted, greaterThan(3000L));
+    }
+
+    @Test public void testPutIntoBackgroundWithoutRestore() {
+        final long msStarted = System.currentTimeMillis();
+        driver.runAppInBackground(-1);
+        assertThat(System.currentTimeMillis() - msStarted, lessThan(1000L));
```

Comment:
Sure, this value, actually, depends on how fast the machine under test is. And Travis, as far as we can see, is quite slow :(